nf_core/pipeline-template/.github/workflows/branch.yml

```diff
@@ -22,7 +24,7 @@ jobs:
         uses: mshick/add-pr-comment@b8f338c590a895d50bcbfa6c5859251edc8952fc # v2
         with:
           message: |
-            ## This PR is against the `master` branch :x:
+            ## This PR is against the `main`/`master` branch :x:
```

Comment:
could we use `GITHUB_REF_NAME ` to know which branch is it?

Reply:
very clever!